.changes/unreleased/NOTES-20250521-145307.yaml

```diff
@@ -0,0 +1,6 @@
+kind: NOTES
+body: 'all: To prevent compilation errors, ensure your Go module is updated to at least terraform-plugin-framework@v1.15.0, terraform-plugin-mux@v0.20.0, terraform
```

Comment:
```suggestion
body: 'all: To prevent compilation errors, ensure your Go module is updated to at least terraform-plugin-framework@v1.15.0, terraform-plugin-mux@v0.20.0,
```